build/musicbrainz-dev/Dockerfile

```diff
@@ -26,6 +26,16 @@ RUN curl -sSL https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -
         gettext \
         g++ \
         git \
+        language-pack-de \
+        language-pack-el \
+        language-pack-es \
+        language-pack-et \
+        language-pack-fi \
+        language-pack-fr \
+        language-pack-it \
+        language-pack-ja \
+        language-pack-nl \
+        language-pack-sq \
```

Comment:
Is it useful to install so many of them? Just having a few of the most complete ones is probably enough for testing.

Reply:
Yes, these are the few most complete ones available in MusicBrainz [`tests`](https://github.com/metabrainz/musicbrainz-server/blob/v-2020-03-16/docker/Dockerfile.tests#L35-L44) image for CI, and in [`website`](https://github.com/metabrainz/musicbrainz-server/blob/v-2020-03-16/docker/templates/macros.m4#L128) image for [test](https://test.musicbrainz.org) and [beta](https://beta.musicbrainz.org) (but `sq`) websites. It can be useful to maintainer for debugging an error spotted from any of these, or just to developer for testing with favorite language.

IIRC, installing these language packs must be done ahead of installing some Node or Perl modules, that is the main reason to install these packs here rather than carry developer with the burden of maintaining a custom `Dockerfile` and reinstalling Node and Perl modules if needed.

Ideally, languages should be selectable through Docker environment, so you can just install the ones you want to work with. Reported as MBVM-50.